azkaban-db/src/main/sql/upgrade.3.80.0.to.3.81.0.sql

```diff
@@ -0,0 +1,5 @@
+-- DB Migration from release 3.80.0 to 3.81.0
+-- PR #2363 Implements thin archive support. The thin archive json text must be stored as a blob along with the
+-- project version.
+--
+ALTER TABLE project_versions ADD COLUMN startup_dependencies MEDIUMBLOB DEFAULT NULL;
```

Comment:
Did you run this locally? I Am not sure if it's "ADD COLUMN" or just "ADD".

Reply:
Also: do we want to create `validated_dependencies` here, or do you mean to run this *and* `create.validated_dependencies.sql`, separately?

Reply:
It appeared that the precedent was that new tables were created separately and not included in the migration.

Reply:
Also yeah, whoops - the query is definitely wrong :)